indico/modules/categories/client/js/calendar.js

```diff
@@ -68,7 +68,7 @@ import CalendarLegend from './components/CalendarLegend';
                 ignoreClick = true;
               }
             };
-            injectModal(
+            return injectModal(
```

Comment:
I don't think that return is needed

Reply:
`async` functions always return a promise. If this one is not returned then a dummy one will be allocated and implicitly returned, so I thought it'd be very slightly more efficient.

Reply:
I'd say it does not make much sense to return something that's not used by the caller.. it implies to someone reading it the return value is used for *something*

Reply:
I don't really agree with that, since functions should work regardless of the caller and what it's doing with the return type. Still, I added [d493d9e](https://github.com/indico/indico/pull/6110/commits/d493d9e731c52440282e93698a89978cab4c0f2c) with the changes.